cycle_en_terre_website_design/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# -*- encoding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Cycle en terre Website design',
+    'category': 'Website',
+    'summary': 'Design modification to look like their own WordPress Website '
+               'http://www.cycle-en-terre.be',
+    'website': 'http://www.cycle-en-terre.be',
```

Comment:
In the 'website' key, put the url to the github repository. So here : 
```
'website': 'https://github.com/coopiteasy/cycle-en-terre'
```

Reply:
Done

Reply:
Done